pages/ksphere/kommander/latest/projects/index.md

```diff
@@ -43,3 +43,11 @@ Using project ConfigMaps and project Secrets you can define configuration resour
 ## Platform Services
 
 Platform Services provide a catalog of applications you can deploy across all project clusters. See [Addon Catalog](/ksphere/kommander/latest/projects/addon-catalog) for more information
+
+## Quotas
+
+Quotas can be set to constrain resource usage. Quotas are applied to all project clusters.
+
+Kommander provides a set of default resources you can set Quotas for, or you can define custom Quotas. It is recommended to set Quotas for CPU and Memory.
```

Comment:
For me, "Quotas for custom resources" sounds more descriptive than "custom Quotas".

Reply:
I agree. I'll update it 👍